src/core/evaluator.js

```diff
@@ -1459,10 +1459,11 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
               var isSameTextLine = !textState.font ? false :
                 ((textState.font.vertical ? args[0] : args[1]) === 0);
               advance = args[0] - args[1];
+              var relativeAdvance = advance - textContentItem.lastAdvanceWidth - textContentItem.lastAdvanceHeight;
```

Comment:
Looks like it's `diff` that we are calculating below. Also shall we calculate `advance` this way (instead of introducing relative advance)? Does it require resetting textContentItem.lastAdvanceWidth/lastAdvanceHeight to different value?